README.md

```diff
@@ -100,13 +100,13 @@ import * as module from "./module";
 console.log(module.inner.a);
 ```
 
-In this example the export `b` can be removed in production mode.
+In this example, the export `b` can be removed in production mode.
 
 (since alpha.15)
 
 ## Inner-module tree-shaking
 
-webpack 4 didn't analyse dependencies between exports and imports of an module. webpack 5 has a new option `optimization.innerGraph`, which is enabled by default in production mode, that runs a analysis on symbols in a module to figure out dependencies from exports to imports.
+webpack 4 didn't analyze dependencies between exports and imports of an module. webpack 5 has a new option `optimization.innerGraph`, which is enabled by default in production mode, that runs an analysis on symbols in a module to figure out dependencies from exports to imports.
```

Comment:
`analyze` vs `analyse`. The US English vs British English. Not actually a typo, but a lot of people around the world learn British English grammar and speaks US English. 

I leave this to @sokra